### ISSUE TITLE
IsActive() returns null and invalid

### ISSUE THREAD

**Issue by mstroppel** [zapadi/vies-dotnet#13]
It seems response of Vies API has changed and includes namespaces now:

```xml
<env:Envelope xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">
<env:Header/>
  <env:Body>
    <ns2:checkVatResponse xmlns:ns2="urn:ec.europa.eu:taxud:vies:services:checkVat:types">
      <ns2:countryCode>DE</ns2:countryCode>
      <ns2:vatNumber>312635404</ns2:vatNumber>
      <ns2:requestDate>2022-08-15+02:00</ns2:requestDate>
      <ns2:valid>true</ns2:valid>
      <ns2:name>---</ns2:name>
      <ns2:address>---</ns2:address>
    </ns2:checkVatResponse>
  </env:Body>
</env:Envelope>
```

So this code does no longer work:

https://github.com/zapadi/vies-dotnet/blob/151b95a0510f375dd34fe53ead413fc8200df06d/src/vies/ViesManager.cs#L367 

Suggestions:

- change to `<ns2:valid>....</vs2:valid>`
- parse XML

**Comment by Macs-cyber** [comment]
Also in the enum [EUCountryCode.cs](https://github.com/zapadi/vies-dotnet/blob/151b95a0510f375dd34fe53ead413fc8200df06d/src/vies/EUCountryCode.cs)
``` C#
internal enum EuCountryCode {}
```
There is still Great Britain which is no longer part of the UE...😅

**Comment by sygmond** [comment]
@Macs-cyber as long as the vat validation works also for GB... is still useful to have it in there :P

**Comment by mstroppel** [comment]
@sygmond that would be great ❤

**Comment by zapadi** [comment]
> Also in the enum [EUCountryCode.cs](https://github.com/zapadi/vies-dotnet/blob/151b95a0510f375dd34fe53ead413fc8200df06d/src/vies/EUCountryCode.cs)
> 
> ```cs
> internal enum EuCountryCode {}
> ```
> 
> There is still Great Britain which is no longer part of the UE...😅

We'll keep it for now. Who knows...maybe they will come back 😆

**Comment by sygmond** [comment]
Multumesc @zapadi :P
This issue sounds like being resolved (till the soap service get's changed again).
Zapadi, will this go automatically to nuget.org or it has to be pushed manually?

**Comment by Macs-cyber** [comment]
> @Macs-cyber as long as the vat validation works also for GB... is still useful to have it in there :P @mstroppel I will submit a pull request with your fix, if that's ok by you.

Not quite, if u try to send a request with "GB" country code u will receive an INVALID_INPUT response from the server...

**Comment by sygmond** [comment]
Aha. I will try it later in the evening.
But as far as I noticed, the UnitTest is testing also against a GB vat number and in my last PR all tests passed.

**Comment by Macs-cyber** [comment]
I don't know what to tell you, probably also the UnitTest are not updated... honestly i trust more the actual response of the server... I simulated it with postman and with all the other country codes it works perfectly!

**Comment by zapadi** [comment]
Problem solved - v2.0.0 deployed
